pkg/pillar/cmd/zedmanager/handleverifier.go

```diff
@@ -252,27 +260,75 @@ func handleVerifyImageStatusDelete(ctxArg interface{}, key string,
 	log.Infof("handleVerifyImageStatusDelete done for %s\n", key)
 }
 
-// check whether the cert files are installed
-func checkCertsForObject(safename string, ss *types.StorageStatus) bool {
-
-	var cidx int = 0
-
+// whether the object contains certs reference
+func containsCerts(safename string, ss *types.StorageStatus) bool {
+	cidx := 0
 	// count the number of cerificates in this object
 	if ss.SignatureKey != "" {
 		cidx++
 	}
-	for _, certUrl := range ss.CertificateChain {
-		if certUrl != "" {
+	for _, certURL := range ss.CertificateChain {
```

Comment:
type StorageStatus struct {
     CertificateChain   []string

   --> Can you have a null string here? Shouldn't we assert here?
   if certURL == "" {
      log.Fatalf("Invalid Certificate - Empty string. indx: %d", indx)
   }

Reply:
done.